srcpkgs/openttd/INSTALL.msg

```diff
@@ -0,0 +1,6 @@
+OpenGFX, OpenSFX and OpenMSX have been moved to separate packages. Please
+install openttd-opengfx, openttd-opensfx and openttd-openmsx packages
+respectively if you need them.
+
+Now these packages are installed automatically. This would change in the
+future.
```

Comment:
```suggestion
The openttd package automatically installs the add-on packages openttd-opengfx,
openttd-opensfx and openttd-openmsx. This will change in a future update. Mark 
the packages manually installed if you wish to continue using these add-ons.
```